R/drive_list.R

```diff
@@ -0,0 +1,162 @@
+#' List files on Google Drive
+#'
+#' @param path character vector, path name(s) where the Google drive files are
+#'   that you would like to list. Defaults to the "My Drive" directory.
+#' @param pattern character vector, regular expression(s) of title(s) of
```

Comment:
I know I am probably the source of this, because I accepted multiple regexes in `gs_ls()`. In hindsight, I think it's preferable to play more dumb and only accept **a** regex. If people want to catenate multiple regexes, they can do this themselves. Plus, in this function, the whole power of `q` is just one argument away!

I would be like `pattern` in `list.files()` and accept only one regex.